changelog/1223.breaking.rst

```diff
@@ -0,0 +1 @@
+:attr:`Emoji.guild_id` can now be ``None`` if the emoji is owned by an application. You can use :meth:`Emoji.is_app_emoji` to check for that.
```

Comment:
If its already breaking then AppEmoji makes the most sense. it can be a subclass with guild_id of None if that were to help things.

Reply:
What if we hack it with guild id of zero?